src/Nethermind/Nethermind.Mev.Test/Tests/single_gas_high_0_mev.json

```diff
@@ -17,6 +17,8 @@
       ]
     }
   ],
-  "optimalProfit" : 2100000000000000,
+  "optimalProfitV1" : 2100000000000000,
+  "optimalProfitV2_max1bundles" : 2100000000000000,
```

Comment:
remove, there is only one optimal profit, these tests are to show which selector is the closest to optimal